apps/mdn/mdn-aws/infra/modules/multi_region/rds/main.tf

```diff
@@ -82,6 +82,43 @@ resource "aws_db_instance" "mdn_rds" {
   tags                         = merge({ "Name" = "MDN-rds-${var.environment}" }, local.tags)
 }
 
+resource "aws_db_instance" "mdn_postgres" {
+  count = var.enabled ? 1 : 0
+
+  allocated_storage           = var.postgres_storage_gb
+  allow_major_version_upgrade = var.postgres_allow_major_version_upgrade
+  auto_minor_version_upgrade  = var.postgres_auto_minor_version_upgrade
+  backup_retention_period     = var.postgres_backup_retention_days
+  backup_window               = var.postgres_backup_window
+
+  db_subnet_group_name = element(aws_db_subnet_group.rds.*.name, count.index)
+
+  depends_on = [aws_security_group.mdn_rds_sg]
+  engine     = var.postgres_engine
+
+  iops = var.environment == "prod" ? 1000 : null
+
+  engine_version               = var.postgres_engine_version
+  identifier                   = var.postgres_identifier #"mdn-prod-postgres"
+  instance_class               = var.postgres_instance_class
+  maintenance_window           = var.postgres_maintenance_window
+  monitoring_interval          = var.monitoring_interval
+  multi_az                     = true
+  name                         = var.postgres_db_name #"developer_mozilla_org"
+  parameter_group_name         = "default.postgres13" # need to create this state as well
+  password                     = var.postgres_password
+  publicly_accessible          = false
+  storage_encrypted            = var.postgres_storage_encrypted
+  storage_type                 = var.postgres_storage_type
+  username                     = var.postgres_username
+  performance_insights_enabled = true
+  skip_final_snapshot          = false
+  port                         = var.postgres_port
+  vpc_security_group_ids       = [aws_security_group.mdn_rds_sg[0].id]
+  final_snapshot_identifier    = "mdn-prod-postgres-final"
```

Comment:
should this name for this object be prod for the stage postgres?

Reply:
Great :eye: @duallain , pushed up a fix in a3803cc